src/noit_check.c

```diff
@@ -1611,7 +1643,7 @@ check_recycle_bin_processor(eventer_t e, int mask, void *closure,
 }
 
 int
-noit_poller_deschedule(uuid_t in, mtev_boolean log) {
+noit_poller_deschedule(uuid_t in, mtev_boolean log, mtev_boolean readding) {
```

Comment:
This supposed to be "reading"?

Reply:
heh.. no.. "re adding"  this is true if we're descheduling before we go an add it back in.